lib/config/a11y.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
     'no-abstract-roles': 'error',
     'no-accesskey-attribute': 'error',
     'no-aria-hidden-body': 'error',
-    'no-down-event-binding': 'error',
+    'no-pointer-down-event-binding': 'error',
```

Comment:
Should probably move down to respect the order of this list. Same goes for the `recommended` config file.

Reply:
Good catch, will alpha sort this